test/IECoreGL/TextureLoaderTest.py

```diff
@@ -56,6 +56,32 @@ def testLoadGreyscale( self ) :
 		t = l.load( "test/IECoreImage/data/jpg/greyscaleCheckerBoard.jpg" )
 		self.failUnless( isinstance( t, IECoreGL.LuminanceTexture ) )
 
+	def testMaximumTextureResolution( self ) :
+
+		maxResolution = 128
+
+		l = IECoreGL.TextureLoader( IECore.SearchPath( "./" ) )
+
+		t = l.load( "test/IECoreImage/data/exr/carPark.exr" )
+		i = t.imagePrimitive()
+		size = i.dataWindow.size()
+
+		self.assertTrue( max( size.x, size.y ) > maxResolution )
+
+		t = l.load( "test/IECoreImage/data/exr/carPark.exr", maxResolution )
+		i = t.imagePrimitive()
+		size = i.dataWindow.size()
+
+		# Test if resolution is adhering to our max resolution and has a reasonable size.
+		self.assertTrue( max( size.x, size.y ) <= maxResolution )
```

Comment:
`assertLessEqual()` and `assertGreater()` give better diagnostics when they fail - it's worth getting into the habit of using them.

Reply:
Also much better, thanks :) 7ced655